enhancements/oc/day2-add-nodes.md

```diff
@@ -0,0 +1,271 @@
+
+## Motivation
+The task of adding a node to an already existing cluster (without automatic scaling) still remains a complex task for the end user, with 
+many different methods available that in some cases may lead to custom automations difficult to maintain. In general, scaling up an on-premises 
+cluster should be a simple operation, mostly automated, for the convenience of the end user, using the already available interfaces for 
+cluster administration.
+
+Currently the agent-based installer method significantly improved the user experience for deploying a cluster - especially in disconnected 
+scenarios - by focusing on full automation of the OpenShift-specific parts of the process, and leaving to the user the decision to automate 
+or not the required cluster infrastructure. The approach addressed several pain points, in particular by not requiring any additional provisioning 
+hardware, and by applying a number of live pre-flight validations in the target environment to spot in advance those issues that could hamper 
+the setup. Also, capturing all the necessary elements for the installation in a single ISO image simplified the application and distribution 
+into different environments. 
+
+The current enhancement proposes a standard and unified way for adding nodes to an on-premises cluster, by replicating the same approach 
+successfully adopted for the installation process, so that the end user could benefit from a more consistent and simple workflow. In addition, 
+the standardization will make obsolete the hand-rolled UPI method, the need to use MCE to manage local clusters, and (unsupported) field-developed 
+automations that are expensive to maintain.
+
+### User Stories
+- As a cluster administrator, I want to add (or replace) one or more nodes to an existing cluster (potentially deployed in a disconnected environment) 
+  by generating an ISO and use it to boot the required nodes
+- As a cluster administrator, I want to provide the initial static network configuration required for each different host that will be added to the cluster
+- As a cluster administrator, I want to validate the nodes to be added before initiating the procedure and be informed of any validation failures
+- As a cluster administrator, I want to be able to monitor the process of adding new nodes
+
+### Goals
+- Allow the user to easily add one or more worker nodes to an existing (disconnected) cluster where MachineSet scaleout is not available or feasible, 
+  with the minimum amount of configuration requested
+- Provide a simpler interface by reusing existing command line tools for cluster administration
+- When adding multiple nodes, a failure in adding a node should not impact the other nodes currently involved in the process
+- Cover at least Baremetal, vSphere, Nutanix and None platform (regardless of the installation method used)
+- For IPI clusters, provide support for the scenario where the user can’t use the hosts BMCs
```

Comment:
To clarify because it was in its own bullet point, to support IPI clusters where users can't use the hosts' BMCs, the workflow isn't any different compared to the other platforms being covered.